pkg/report/template.go

```diff
@@ -39,6 +44,9 @@ func Generate(dibReport Report, dag dag.DAG) error {
 		return fmt.Errorf("unable to render report templates: %w", err)
 	}
 
+	finalReportURL := printReportURL(dibReport)
+	logrus.Infof("success, you can now browse report from \"%s\"", finalReportURL)
```

Comment:
We may change the wording here as the report is generated even if the build is not successful: 

```suggestion
	logrus.Infof("Generated HTML report: \"%s\"", finalReportURL)
```